src/routes/learn/mlp-training.svx

```diff
@@ -0,0 +1,232 @@
+1. the flexibility of the MLP objects make them generally more capable of learning complex relationships between inputs and outputs, 
+2. the MLP objects involve more parameters and will take much longer to `fit` (aka. train) than the KNN objects, and 
+3. the KNN objects will likely take longer to make predictions than the MLP objects, depending on the size of the dataset (although they're still quite quick!).
+
+## The Training Process
+
+### Feed-forward
+
+The internal structure of a multi-layer perceptron (MLP) creates a "feedforward" network in which input values are received at input neurons, passed through sequential hidden layers of neurons which compute values that are passed on to the following layer, eventually arriving at output neurons which provide their values as the output of the neural network. These sequential layers are "fully connected", meaning that each neuron in one layer connects to every neuron in the following layer. These connections are _weights_ that are multiplied by the values to control how much each neuron's output influences the neurons it is connected to. The neurons (except in the input layer) also use a bias (an added offset) and an [activation function](/#activation-functions) to compute its output. The weights and biases are internal parameters that are not controlled at all by the user. 
+
+<Image
+src="/learn/mlp/211220_01_synth_control_sc.txt.jpg"
+label="A multi-layer perceptron neural network that takes two inputs, has one hidden layer of seven neurons, and predicts ten outputs. Each black arrow is a weight. Each of the neurons in the hidden layer and output layer also have a bias."
+/>
+
+### Audio Mixer Metaphor
+
+A useful metaphor for understanding how the feedforward algorithm works is to think of a neural network like an audio mixer. The inputs to the neural network are like the audio inputs to the mixer. The weights are like volume adjustments that control how much of an input signal gets passed on (in the case of a neural network, passed on to the next layer). The biases are like [DC offsets](https://manual.audacityteam.org/man/dc_offset.html#:~:text=DC%20offset%20is%20a%20mean,and%20how%20to%20remove%20it.) that just shift the signal up or down. [Activation functions](/#activation) are like distortion--they introduce some non-linearities into the signal (which for neural networks help them learn non-linear relationships between the inputs and outputs). Lastly, the outputs of the neural network are like the outputs of the audio mixer--a volume adjusted, DC offset, and distorted combination of all the inputs.
```

Comment:
`[Activation functions](/#activation) are like distortion--they introduce some non-linearities into the signal (which for neural networks help them learn non-linear relationships between the inputs and outputs).`

Needs a space and I wouldn't use -- but instead – (and not -) which are distinct.